src/figure_factory.py

```diff
@@ -10,14 +10,14 @@ class FigureFactory:
     __kind_randomizer = FigureKindRandomizer()
 
     @staticmethod
-    def create(kind: FigureKind) -> Figure:
+    def create(kind: FigureKind, weight: int) -> Figure:
         start_rotation = RotationKind.ROTATION_0
         figure_points = FigureMover.get_points_after_rotation(kind, start_rotation)
 
-        return Figure(kind, Point(0, 0), figure_points, start_rotation)
+        return Figure(kind, Point(0, weight // 2 - 2), figure_points, start_rotation)
```

Comment:
extract `weight // 2 - 1` (correct to `- 1`) to variable `center_position`

Reply:
not done

Reply:
done